opentelemetry/instrumentation/app/server.go

```diff
@@ -28,11 +27,9 @@ import (
 // the number of milliseconds slept as its response.
 // [START opentelemetry_instrumentation_handle_single]
 func handleSingle(w http.ResponseWriter, r *http.Request) {
```

Comment:
I wonder if we should put this on the handleMulti endpoint so we only add one extra span?

Reply:
Based on the position of this comment, I'm not sure what you're referring to. 

Are you suggesting to remove custom span generation from `doWork` to `handleMulti` ?

Reply:
If you look at the example trace, IIRC, it currently can have up to 1 server span for `/multi `+ 14 spans for single (client + server).  So adding a new span to `/single` will add up to 7 new spans to the trace, which will make it visually much larger.  If you add a new span to `/multi`, it will only add a single span to the trace.

I'm wondering if up to 22 spans will make the trace seem cluttered.

Reply:
I see, I can certainly remove the custom span from `/single` to keep things simple. I think a single custom span in `/multi` should be good enough to demonstrate custom instrumentation for tracing.